ivy/functional/ivy/losses.py

```diff
@@ -81,24 +81,100 @@ def binary_cross_entropy(true, pred, epsilon=1e-7):
     return -(ivy.log(pred) * true + ivy.log(1 - pred) * (1 - true))
 
 
-def sparse_cross_entropy(true, pred, axis=-1, epsilon=1e-7):
+def sparse_cross_entropy(
+        true: Union[ivy.Array, ivy.NativeArray],
+        pred: Union[ivy.Array, ivy.NativeArray],
+        axis: Optional[int] = -1,
+        epsilon: Optional[float] = 1e-7,
+) -> ivy.Array:
     """Computes sparse cross entropy between logits and labels.
 
     Parameters
     ----------
     true
-        True labels as logits.
+        input array containing the true labels as logits.
```

Comment:
I think the word **logits** makes it confusing. The description for `true` should instead be `input array containing true labels`

Reply:
Generally ground truth is passed as ith categorical number. There is no sense of logits in ground truth. I agree with the logic, but I didn't want to change the initial wording.

Reply:
Not always. If you are using  categorical crossentropy instead of sparse categorical crossentropy, you need to pass labels as probabilities (or one hot encodings).

Reply:
You are right. The word `logits` even though it doesn't make sense but creates confusion. I think it should be removed. What do you think?